bin/grep.py

```diff
@@ -9,6 +9,7 @@
 import sys
 
 def main(args):
+    global _stash
```

Comment:
This is not needed - `_stash` is never assigned inside the function, so all uses of `_stash` already refer to the global name by default.

Reply:
I actually like this approach and using _explicit is better than implicit_ as a justification, I am not inclined to change it unless __from stash import \_stash__ works.